imports/ui/components/Card.jsx

```diff
@@ -119,7 +119,10 @@ class CardComponent extends Component {
               </div>
             )}
           </div>
-          {this.props.currentUser && <AddShoppingList item={this.state.data} />}
```

Comment:
adding `this.props.currentUser` should fix the regression of user need to refresh to see new shopping list